tests/cxx/built_ins_template.cc

```diff
@@ -0,0 +1,20 @@
+//===--- built_ins_template.cc - test input file for iwyu -----------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+//Built-ins have no declaration, thus they need no include.
+
+__type_pack_element<0, int> t;
+
+//TODO: Ignoring use of __type_pack_element ... built-in function
```

Comment:
Do you mean you'd like to check the IWYU output here? I wonder if it's possible at the high verbosity level it is right now... Maybe just skip the TODO.